src/scripts/clientUtils.js

```diff
@@ -29,20 +29,43 @@ export async function serverAddress() {
     // if (current) return Promise.resolve(current);
 
     const urls = [];
+    if (window.location.href.indexOf('/web/') !== -1) { // Do we polyfill String.prototype.includes?
+        const split = window.location.href.split('/web/');
+
+        for (let i = split.length - 1; i ; i--) {
+            urls.push(split.slice(0, i).join('/web/'));
+        }
+    }
```

Comment:
I feel like this section is more complicated than it needs to be..

```js
const urlLower = window.location.href.toLowerCase();
const index = urlLower.lastIndexOf('/web');
if (index != -1) {
    urls.push(urlLower.substring(0, index));
}
```

I don't know why we are looping. In my opinion the last /web should suffice.

Reply:
looping in case someone had the silly idea of including `/web/` in their baseurl